class BaseException {
	message(message) {
		return console.log(this, message);
	}
}
